From bb707a0633d6d6cbb456b1ebef7a73f869ec5fdd Mon Sep 17 00:00:00 2001
From: Ethan Chien <liangtie.qian@gmail.com>
Date: Sat, 24 Aug 2024 00:01:47 +0800
Subject: [PATCH] Change wxDEFAULT_TOOLKIT to qt on WIN32

---
 build/cmake/toolkit.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/cmake/toolkit.cmake b/build/cmake/toolkit.cmake
index c5febca..2a9db05 100644
--- a/build/cmake/toolkit.cmake
+++ b/build/cmake/toolkit.cmake
@@ -19,7 +19,7 @@ if(CMAKE_OSX_SYSROOT MATCHES iphoneos)
 endif()
 
 if(WIN32)
-    set(wxDEFAULT_TOOLKIT msw)
+    set(wxDEFAULT_TOOLKIT qt)
     set(wxTOOLKIT_OPTIONS msw gtk2 gtk3 qt)
     set(wxPLATFORM WIN32)
 elseif(APPLE AND IPHONE)
-- 
2.45.0.windows.1

